web/src/beta/features/WorkspaceSetting/index.tsx

```diff
@@ -6,38 +6,54 @@ import CursorStatus from "../CursorStatus";
 import useProjectsHook from "../Dashboard/ContentsContainer/Projects/hooks";
 
 import useWorkspaceHook from "./hooks";
+import Members from "./innerPages/Members/Members";
 import Workspace from "./innerPages/Workspaces/Workspaces";
 
 type Props = {
   tab: string;
   workspaceId?: string;
 };
 
+enum TABS {
+  WORKSPACE = "workspace",
+  MEMBERS = "members"
+}
+
 const WorkspaceSetting: FC<Props> = ({ tab, workspaceId }) => {
   const {
     handleFetchWorkspaces,
     handleUpdateWorkspace,
-    handleDeleteWorkspace
+    handleDeleteWorkspace,
+    handleAddMemberToWorkspace,
+    handleSearchUser,
+    handleUpdateMemberOfWorkspace,
+    handleRemoveMemberFromWorkspace
   } = useWorkspaceHook();
 
   const { filtedProjects } = useProjectsHook(workspaceId);
 
   const { tabs } = useAccountSettingsTabs({ workspaceId: workspaceId ?? "" });
 
   return (
-    <>
-      <SettingBase tabs={tabs} tab={tab} workspaceId={workspaceId}>
-        {tab === "workspace" && (
-          <Workspace
-            handleFetchWorkspaces={handleFetchWorkspaces}
-            handleUpdateWorkspace={handleUpdateWorkspace}
-            handleDeleteWorkspace={handleDeleteWorkspace}
-            projectsCount={filtedProjects?.length}
-          />
-        )}
-      </SettingBase>
+    <SettingBase tabs={tabs} tab={tab} workspaceId={workspaceId}>
+      {tab === TABS.WORKSPACE && (
+        <Workspace
+          handleFetchWorkspaces={handleFetchWorkspaces}
+          handleUpdateWorkspace={handleUpdateWorkspace}
+          handleDeleteWorkspace={handleDeleteWorkspace}
+          projectsCount={filtedProjects?.length}
+        />
+      )}
+      {tab === TABS.MEMBERS && (
+        <Members
+          handleSearchUser={handleSearchUser}
+          handleAddMemberToWorkspace={handleAddMemberToWorkspace}
+          handleUpdateMemberOfWorkspace={handleUpdateMemberOfWorkspace}
+          handleRemoveMemberFromWorkspace={handleRemoveMemberFromWorkspace}
+        />
+      )}
       <CursorStatus />
-    </>
+    </SettingBase>
```

Comment:
Please still locate `CorsorStatus` besides `SettingBase`, as a child will lead to incorrect position for the indicator.
Please help update the same structure on `AccountSettings` as well.